docs/admin/disruptions.md

```diff
@@ -0,0 +1,87 @@
+---
+assignees:
+- mml
+
+---
+This guide is for anyone wishing to specify safety constraints on pods or anyone
+wishing to write software (typically automation software) that respects those
+constraints.
+
+* TOC
+{:toc}
+
+## Rationale
+
+Various cluster management operations may "voluntarily" evict pods. By
+"voluntary" we mean the operation can be safely delayed for a reasonable period
+of time. The principal examples today are draining a node for maintenance or
+upgrade (`kubectl drain`), and cluster autoscaling down. In the future we will
+have the
+[rescheduler](https://github.com/kubernetes/kubernetes/blob/master/docs/proposals/rescheduling.md)
+and possibly other examples. (In contrast, something like evicting pods because
+a node has become unreachable or reports NotReady, is not "voluntary.") For
```

Comment:
I'd start a new paragraph here.

"For voluntary evictions, it can be useful..."